pyvista/plotting/_property.py

```diff
@@ -230,6 +237,16 @@ def __init__(
         self.line_width = line_width
         if culling is not None:
             self.culling = culling
+        if vtk_version_info < (9, 3) and edge_opacity is not None:  # pragma: no cover
+            from pyvista.core.errors import VTKVersionError
+
+            raise VTKVersionError(
+                "`edge_opacity` cannot be used under VTK v9.3.0. Try installing VTK v9.3.0 or newer."
+            )
```

Comment:
I'm thinking this should be a warning rather than an error?

Reply:
```suggestion
        if vtk_version_info < (9, 3) and edge_opacity is not None:  # pragma: no cover
            import warnings

            warnings.warn(
                '`edge_opacity` cannot be used under VTK v9.3.0. Try installing VTK v9.3.0 or newer.',
                UserWarning,
            )
```